src/app/inventory/store/stats.ts

```diff
@@ -297,33 +318,43 @@ function buildStat(
     additive:
       statDef.statCategory === DestinyStatCategory.Defense &&
       statDef.aggregationType === DestinyStatAggregationType.Character,
+    isConditionallyActive: itemStat.isConditionallyActive,
   };
 }
 
 function enhanceStatsWithPlugs(
   itemDef: DestinyInventoryItemDefinition,
   stats: DimStat[], // mutated
-  statsByHash: { [k: number]: DimStat }, // mutated
```

Comment:
I thought we mutated this to use in other places?

Reply:
Apparently not, it is only used within enhanceStatsWithPlugs. From my understanding mutating the stats array should also mutate statsByHash as the DimStat objects should be shared across the two?